README.adoc

```diff
@@ -385,6 +385,24 @@ class Transaction < ActiveRecord::Base
 end
 ----
 
+=== Enums [[enums]]
+
+Prefer hash syntax over arrays for `enums`.
```

Comment:
WDYT of
```
`enum`s
```
or 
Prefer using the hash syntax for `enum`.

Reply:
Changed to `Prefer using the hash syntax for enum.`